client/rpcserver/handlers.go

```diff
@@ -581,15 +581,18 @@ var helpMsgs = map[string]helpMsg{
 	},
 	newWalletRoute: {
 		pwArgsShort: `"appPass" "walletPass"`,
-		argsShort:   `assetID ("config")`,
+		argsShort:   `assetID ("path" "settings")`,
 		cmdSummary:  `Connect to a new wallet.`,
 		pwArgsLong: `Password Args:
     appPass (string): The DEX client password.
     walletPass (string): The wallet's password.`,
 		argsLong: `Args:
     assetID (int): The asset's BIP-44 registered coin index. e.g. 42 for DCR.
       See https://github.com/satoshilabs/slips/blob/master/slip-0044.md
-    config (string): Optional. The path to the wallet's config file.`,
+     path (string): Optional. The path to a configuration file.
+	 settings (string): A JSON-encoded string->string mapping of additional
+       configuration settings. These settings take precedence over any settings
+       parsed from file`,
```

Comment:
Makes sense, but the formatting is a little funny because there are mixed TABs and spaces.  I'm not actually sure what we're going for in this file, but I think space produce the most consistent display formatting.

Reply:
I also wouldn't object to a minimal sample `settings` like `{"account":"default"}` or whatever would be most helpful.